packages/sdk/src/sdk/services/Solana/programs/PaymentRouterClient/PaymentRouterClient.ts

```diff
@@ -87,14 +86,12 @@ export class PaymentRouterClient {
       sourceWallet,
       false
     )
-    const amount = mintFixedDecimalMap[token](args.total)
-    return createTransferCheckedInstruction(
+    const amount = args.total
+    return createTransferInstruction(
```

Comment:
any reason this doesn't need to be checked anymore?

Reply:
how would it be checked?

Reply:
`createTransferChecked` uses the decimals you pass it and compares that against the decimals that the mint is configured to. Before, we knew ahead of time what each mint's decimal count was as they were all preconfigured, but arguably we weren't really benefitting from using createTransferChecked since we were automatically setting the correct decimals instead of requiring the caller to specify